Source/LightboxController.swift

```diff
@@ -325,6 +325,16 @@ open class LightboxController: UIViewController {
     overlayView.resizeGradientLayer()
   }
 
+  fileprivate func calcScrollViewFrame(_ size: CGSize) -> CGRect {
+    let headerVisible = headerView.alpha == 1
+    let headerHeight = headerView.frame.minY + headerView.frame.height
```

Comment:
I think this variable should be called headerMaxY and I believe you can just use headerView.frame.maxY